unit-tests/src/test/scala/utils/ThrowsHelper.scala

```diff
@@ -12,4 +13,11 @@ object ThrowsHelper {
     val c = cond(expectThrows(expectedThrowable, code))
     assert(c)
```

Comment:
Even if I changed this for an `assertEquals` it would not give rich enough error messages, so I decided to add another helper here

Reply:
We really shouldn't use that method or add additional methods to the helper - `@deprecated` probably should have been added to this `object`. This was added because we changed all the tests to JUnit and there were some many tests with this pattern we added the method to ease the transition. Notice the comment. https://github.com/scala-native/scala-native/blob/master/unit-tests/src/test/scala/utils/ThrowsHelper.scala

Even the `assertThrows` from the `AssertThrows` object is planned to be removed as in this issue: https://github.com/scala-native/scala-native/issues/1881

Reply:
Any suggestions to test the exception message?

Reply:
I would suggest calling a helper function that calls `expectThrows` and calls `getMessage` on the exception.
https://github.com/scala-native/scala-native/blob/master/unit-tests/src/test/scala/utils/AssertThrows.scala#L78-L79 Maybe something like this.
```Scala
def exc(s: String): String = {
  val e = expectThrows(classOf[NumberFormatException], decode(s))
  val msg = e.getMessage()
  // could parse off: For input string: "-" to just get the "-" back
  msg
}

assertEquals("""For input string: "-"""", exc("-"))
```
Something like this anyway.

Reply:
That's what I did already by using a new method: `assertThrowsAndMessage` (see below)

Reply:
We need to not add to or use `ThrowsHelper` in new tests so please add your own local helper. Reuse is great but in this case we don't want to use that method. I will be doing a PR to deprecate or try and move it somewhere private.